sockeye/inference.py

```diff
@@ -462,18 +470,21 @@ class LengthPenalty:
     def __init__(self, alpha: float = 1.0, beta: float = 0.0) -> None:
         self.alpha = alpha
         self.beta = beta
-        self.denominator = (self.beta + 1) ** self.alpha
+        self.denominator = float((self.beta + 1.) ** self.alpha)
 
-    def __call__(self, lengths: mx.nd.NDArray) -> mx.nd.NDArray:
+    def __call__(self, lengths: Union[mx.nd.NDArray, int, float]) -> Union[mx.nd.NDArray, float]:
```

Comment:
I hadn't seen the `Union` type before.  That's very useful!

Reply:
yeah, it's a nice feature :)